src/renderer/wg_engine/tvgWgRenderer.cpp

```diff
@@ -324,6 +324,7 @@ bool WgRenderer::target(WGPUSurface surface, uint32_t w, uint32_t h) {
         .usage = WGPUTextureUsage_RenderAttachment,
         .width = w, .height = h,
         #ifdef __EMSCRIPTEN__
+        .alphaMode = WGPUCompositeAlphaMode_Premultiplied,
```

Comment:
this might not be portable among compilers.
```
../src/renderer/wg_engine/tvgWgRenderer.cpp: In member function ‘bool WgRenderer::target(WGPUSurface, uint32_t, uint32_t)’:
../src/renderer/wg_engine/tvgWgRenderer.cpp:332:5: error: designator order for field ‘WGPUSurfaceConfiguration::alphaMode’ does not match declaration order in ‘WGPUSurfaceConfiguration’
  332 |     };
      |     ^
```
please confirm the order of the fields. ( -> alphamode, w, h , presentmode ...)
```
    WGPUChainedStruct const * nextInChain;
    WGPUDevice device;
    WGPUTextureFormat format;
    WGPUTextureUsageFlags usage;
    size_t viewFormatCount;
    WGPUTextureFormat const * viewFormats;
    WGPUCompositeAlphaMode alphaMode;
    uint32_t width;
    uint32_t height;
    WGPUPresentMode presentMode;
```

Reply:
@tinyjin if the "Auto" or "Inherit" having same with the premultiplied, then we'd rather use it.

Reply:
![CleanShot 2024-09-12 at 19 19 57@2x](https://github.com/user-attachments/assets/e4447dec-fc04-4455-9815-f0a195f4f43b)

Auto/Inherit doesn't change to what we expect